File: js/programmers./level_0/특정 문자 제거하기.js
// Lv. 0 특정 문자 제거하기

/**
 * @param {string} my_string
 * @param {string} letter
 * @return {string}
 */

const solution = (my_string, letter) => {
    return my_string
        .split("")
        .filter((v) => v !== letter)
        .join("");
};

File: java/programmers/level_0/src/특정_문자_제거하기.java
public class 특정_문자_제거하기 {

  public static String solution(String my_string, String letter) {
    String answer = "";

    for (int i = 0; i < my_string.length(); i++) {
      if (my_string.charAt(i) != letter.charAt(0)) {
        answer += my_string.charAt(i);
      }
    }
    return answer;
  }

  public static void main(String[] args) {
    System.out.println(solution("abcdef", "f"));
  }

}
